src/components/TextInput/TextInput.stories.tsx

```diff
@@ -1,21 +1,21 @@
 import type { Meta, StoryObj } from '@storybook/react';
-import { TextInput } from '~/src/index';
+import { Button, TextInput } from '~/src/index';
 
 const meta: Meta<typeof TextInput> = {
   title: 'Components (Draft)/Text inputs',
   component: TextInput,
   argTypes: {
-    width: { control: 'select' },
+    fullWidth: { control: 'boolean' },
```

Comment:
Can we rename this `isFull` or `isFullWidth` to match other boolean props naming scheme?